objects/SetupChecker/script.lua

```diff
@@ -1960,21 +1959,39 @@ function setupGainSpiritChoices(obj, choices)
                     spirit = choice.spirit,
                     aspect = choice.aspect,
                 },
-                children = {},
-            })
-        else
-            table.insert(buttons, {
-                tag = "Text",
-                attributes = {
-                    minHeight = "110",
-                },
-                children = {},
             })
         end
     end
+    if count < 4 then
+        table.insert(buttons, {
+            tag = "Text",
+            attributes = {
+                color="#FFFF00",
+                text = "No other spirits available.",
+                fontSize = "44",
+                minWidth = "800",
+                minHeight = "110"
+            }
+        })
+    end
+    if count > 0 then
+        table.insert(buttons, {
+            tag = "Button",
+            attributes = {
+                colors = "#92E5AF|#92E5AF|#72B389|#72B38980",
+                id = "reroll! " .. obj.guid,
+                text = "Reroll Spirit Choices",
```

Comment:
so instead of adding a button to reroll what we could do is just let the player hit the gain a spirit button again to get new options

Reply:
i chatted with MJ about this, and i think the compromise would be to leave the button living alongside the spirit options, but make the button smaller/less emphasized

Reply:
<img width="565" height="346" alt="image" src="https://github.com/user-attachments/assets/339b41d7-c209-483e-a228-761aeb9b3b2f" />

Thoughts on this appearance?  I'll update the PR if this passes muster.

Reply:
sure that looks good, thanks